src/components/BitcoinConnectElement.ts

```diff
@@ -21,6 +21,11 @@ export class BitcoinConnectElement extends InternalElement {
   @state()
   protected _balance: number | undefined = undefined;
 
+  @state()
+  protected _appName: string | undefined = undefined;
+
+  protected appName = 'Bitcoin Connect';
```

Comment:
I don't think we can set a default here, otherwise it'll overwrite anything set on only one of the components. Can you change it to undefined?